### ISSUE TITLE
Explain "experimental" aspect of LocalCudaCluster

### ISSUE THREAD

**Issue by randerzander** [rapidsai/dask-cuda#5]
Dask and cuDF users are starting to use LocalCudaCluster in their work.

This repo's README calls it experimental. Can we better explain what that means so users understand the risks?

**Comment by mrocklin** [comment]
I think that as users play with the library we'll probably learn a lot
about how it should operate.  I would expect API changes in the future and
have no intention of maintaining backwards stability.

Folks are welcome to use it, but we reserve the right to change the API at
any time.  I'm happy to put this statement if preferred.

**Comment by randerzander** [comment]
That makes sense- yes please add that statement to the README.

**Comment by dungthuapps** [comment]
hi, 
Even this is experimental, I tried to play around with GPU as examples but it used only CPU not GPU as I expected.
Here is my example to run
```
    from dask_cuda import LocalCUDACluster
    from dask.distributed import Client
    import dask.array as da

    cluster = LocalCUDACluster()
    client = Client(cluster)
    x = da.random.random((1000000, 1000000), chunks=(1000, 1000))
    y = x + x.T
    z = y[::2, 5000:].mean(axis=1)
    z.compute()
```

My card is RTX-2060 and cuda using is 10.0. In the image, you can se it use only 4MB for python, or said no use at all
![image](https://user-images.githubusercontent.com/22927387/56381434-1f459580-6215-11e9-9485-8966e1b9dc20.png)

All cupy, tensorflow, pytorch has successfully recognized and ran with CUDA, but failed to do it with dask. Even using 
```
dask-scheduler
dask-worker localhost:8786 --resources "GPU=1" 
```
-> still use CPU instead

**Comment by mrocklin** [comment]
@dungthuapps this library doesn't automatically convert your Python code into CUDA enabled code.  It helps in deploying Dask workers on CUDA enabled systems.  I don't know of a solution to the problem that you're looking for.

If you're looking for dask array and cupy in particular, I recommend taking a look at experimental work with cupy: https://blog.dask.org/2019/03/18/dask-nep18

**Comment by dungthuapps** [comment]
@mrocklin 
I am learning how to use dask especially with GPU and firstly tried with the example you wrote here: 
http://matthewrocklin.com/blog/work/2019/01/03/dask-array-gpus-first-steps

However, the example was failed, then I tried to use it in simpler way with simple cuPy. 
the documentation as follows did not help also : 
http://distributed.dask.org/en/latest/resources.html.

I have an RTX 2060 card, is this already CUDA-enabled local system? 
I do not know in which condition I could use dask-cuda, especially for collections. 
Tried with dask.dataframe.read_sql_table(.) also failed with GPU.
